components/script/dom/medialist.rs

```diff
@@ -122,6 +123,7 @@ impl MediaListMethods<crate::DomTypeHolder> for MediaList {
 
     /// <https://drafts.csswg.org/cssom/#dom-medialist-mediatext>
     fn SetMediaText(&self, value: DOMString) {
+        let _rules_modification_scope = RulesModificationScope::new(&self.parent_stylesheet);
```

Comment:
Does this really work? I thought in Rust you can't control when an object is released, which is fully determined by the compiler...